mountpoint-s3/src/inode.rs

```diff
@@ -337,8 +337,15 @@ impl Superblock {
         // Local inode stats never expire, because they can't be looked up remotely
         let stat = match kind {
             // Objects don't have an ETag until they are uploaded to S3
-            InodeKind::File => InodeStat::for_file(0, OffsetDateTime::now_utc(), None, None, None, NEVER_EXPIRE_TTL),
-            InodeKind::Directory => InodeStat::for_directory(self.inner.mount_time, NEVER_EXPIRE_TTL),
+            InodeKind::File => InodeStat::for_file(
+                0,
+                OffsetDateTime::now_utc(),
+                None,
+                None,
+                None,
+                Duration::from_millis(100),
+            ),
+            InodeKind::Directory => InodeStat::for_directory(self.inner.mount_time, Duration::from_millis(100)),
```

Comment:
Let's use the duration specified in `self.inner.cache_config.file_ttl`, etc..

Does this work?

```suggestion
            InodeKind::File => InodeStat::for_file(
                0,
                OffsetDateTime::now_utc(),
                None,
                None,
                None,
                self.inner.cache_config.file_ttl,
            ),
            InodeKind::Directory => InodeStat::for_directory(self.inner.mount_time, self.inner.cache_config.file_ttl),
```

Reply:
One should be dir_ttl, right?

Reply:
oops, yes!